### DISCUSSION TITLE
Welcome to the Community category for Universe 2022!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/events#14] [Community]
💬  Open source is undisputedly the backbone of the world’s software. In this track, you’ll get a better understanding of ways open source impacts you and your business, how you can get involved, and how innersource can break down silos in your own organization and create successful teams. For questions about this space check out the [FAQs](https://github.com/community/events/discussions/1) and don't forget you can [tune in live](http://watch.githubuniverse.com/). Let us know what sessions you're attending or loving below 💟  !

**Comment by mrdiamonddirt** [comment]
attending the keynote and any of the extras i can really been looking forward to this 😃

**Reply by WillOfHeaven** [reply]
Same Here !

**Comment by WillOfHeaven** [comment]
Isn't GitHub copilot specifically Hey GitHub is like OpenAi Codex , Of Course Hey Github is much more intuitive ! :)

**Comment by ClJarvis** [comment]
Keynote, Twitch—a game changer for developers and educators, and some open source sessions.
